io.github.rilsonjoas.MARS.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.rilsonjoas.MARS
+# flathub:--check-url=https://github.com/rilsonjoas/mars
```

Comment:
not sure what you think this is doing, but surely the documentation you are reading is completely wrong.